react-app/src/app/screens/Game/components/Info/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import Button from './components/Button';
+import Info from './layout';
+
+function InfoContainer({ history, status, onClick }) {
+  const movesList = history.map((step, move) => {
+    const text = move ? `Go to move #${move}` : 'Go to game start';
```

Comment:
remember to declare all the string constants on a constant folder, string.js. Declare there strings such as:
```
export const GO_TO_GAME_START = 'Go to game start'
```
And import it here

Reply:
Move all the constant strings to a `string.js` file